followthemoney/mapping/entity.py

```diff
@@ -112,16 +115,24 @@ def map(
         # from that accessible to phone and address parsers.
         for prop in self.properties:
             if prop.prop.type == registry.country:
-                prop.map(proxy, record, entities)
+                discarded_values = prop.map(proxy, record, entities)
+                for value in discarded_values:
+                    log.warn(f"[{self.name}] Discarded unclean value \"{value}\" for property \"{prop.prop.qname}\".")
```

Comment:
Would it make sense to add `proxy.id` to these messages? That might make it a bit easier to debug.

Reply:
Yeah, I originally wanted to do exactly that to make it easier to find the respective row in the source data. Then I realized that some hash of the source data probably isn’t that helpful to a human debugging a mapping…

But… it may still be helpful in case of repeated issues with the same or when using `id_column` which I hadn’t considered before, so I’ll add it.